Scripts/Tracking/Modification/TransformPropertyApplier.cs

```diff
@@ -71,10 +71,15 @@ public class UnityEvent : UnityEvent<EventData>
         [Tooltip("The offset/pivot when applying the transformations.")]
         public GameObject offset;
         /// <summary>
-        /// Determines which axes to apply on when utilising the offset.
+        /// Determines which axes to apply on when utilising the position offset.

```

Comment:
```suggestion
        /// Determines which axes to apply on when utilizing the position offset.
```

Same for the tooltip and the other field+tooltip pair below.